manifests/mod/php.pp

```diff
@@ -34,7 +35,7 @@
   String $template               = 'apache/mod/php.conf.erb',
   Optional[String] $source       = undef,
   Optional[String] $root_group   = $apache::params::root_group,
-  Optional[String] $php_version  = $apache::params::php_version,
+  String[1] $php_version         = $apache::params::php_version,
```

Comment:
Rather than this would it not be better to put a check in to detect when its running on RHEL 9 and fail with a message stating that `php-fpm` should be used instead?

i.e.:
```
  if $facts['os']['name'] == 'Redhat' and $facts['os']['release']['major'] == '9' {
    fail("RHEL 9 does not support 'mod_php' and so 'php-fpm' must be used instead.")
  }
```

Reply:
Make the reasoning and cause explicit?

Reply:
How about this solution: keeping it Optional, but treat `undef` as unsupported?

Reply:
Sound's good to me